alert.go

```diff
@@ -594,6 +599,30 @@ func (a *AlertNode) handleHipChat(hipchat *pipeline.HipChatHandler, ad *AlertDat
 	}
 }
 
+func (a *AlertNode) handleAlerta(alerta *pipeline.AlertaHandler, ad *AlertData) {
+	if a.et.tm.AlertaService == nil {
+		a.logger.Println("E! failed to send Alerta message. Alerta is not enabled")
+		return
+	}
+	err := a.et.tm.AlertaService.Alert(
+		alerta.Token,
+		alerta.Resource,
+		alerta.Event,
+		alerta.Environment,
+		ad.Level.String(),
```

Comment:
I looked at the Alerta Severities as well, since Alerta has many more you should use a switch statement to map the Kapacitor alert levels to the Alerta severity strings since they do not map up by default.

 See handleVictorOps for an example of what I mean.